### PR TITLE
Merge final `version/1.0` branch with `master`

### PR DESCRIPTION
**Pull request by timmyc**
Bringing in the last bits of the version/1.0 branch to master. Once this is merged any future fixes for the `1.0.x` version should land in master and then be cherry picked back to the release branch.

### CODE REVIEW

File: client/dashboard/task-list/tasks/steps/plugins.js [hunk 1]
```diff
@@ -119,10 +128,14 @@ class Plugins extends Component {
 			return null;
 		}
 
-		if ( 0 === pluginSlugs.length ) {
+		if ( pluginSlugs.length === 0 ) {
```
Comment (psealock) [inline client/dashboard/task-list/tasks/steps/plugins.js]: weird, I would have figured it the other way.

**Review by psealock — COMMENTED** [review]
Changes from https://github.com/woocommerce/woocommerce-admin/pull/3848/commits/8d578baf8cb386b56fb68bb3098424e162ec13f7 look legit to me.

>Once this is merged any future fixes for the 1.0.x version should land in master and then be cherry picked back to the release branch.

I'm just wondering if this is easier or why the change?

**Comment by timmyc** [comment]
> I'm just wondering if this is easier or why the change?

I think once we kind of "officially" move on back to dev full time on master, cherrypicking individual changes back to `version/1.0` using `hub` should be pretty easy to do. That is how woo core operates too, so part of it is habit from my experience there, but also might be nice to follow their standard.

**Review by psealock — APPROVED** [review]

**Comment by psealock** [comment]
I'm going to merge this so I can get started on updating the wp.data settings refactor branch, https://github.com/woocommerce/woocommerce-admin/pull/3313

**Pull request merged**